lib/bolt/executor.rb

```diff
@@ -267,12 +267,12 @@ def run_task(targets, task, arguments, options = {})
     end
 
     def upload_file(targets, source, destination, options = {})
-      description = options.fetch('_description', "file upload from #{source} to #{destination}")
+      description = options.fetch('_description', "file or directory upload from #{source} to #{destination}")
```

Comment:
I think this logging/outputting is intended more to map to the action `file upload` rather than describe to the user the behavior of the action. I would lean toward leaving that part out and instead just focus on documenting what to expect with `file upload`.

Reply:
Ok, removed the executor changes